client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -391,6 +392,24 @@ class DeltaSharingRestClient(
     }
   }
 
+  private def logFirst10Lines(
```

Comment:
Call it logKLines and pass some param? The param could be some static value for now.